### PR TITLE
fr-FR: Improve translation

### PR DESCRIPTION
**Pull request by rmnvgr**
- All quotation marks replaced with guillemets
- Text that should be in italics put in small fonts
- Correct spaces around punctuation
- Correct capitalization
- Accents on uppercase characters
- Correct apostrophes
- Correct dashes
- Follow [translation guidelines](https://github.com/OpenRCT2/Localisation/wiki/Translator-guidelines:-fr_FR)

### CODE REVIEW

File: data/language/fr-FR.txt [hunk 1]
```diff
@@ -866,131 +866,131 @@ STR_1476    :{WINDOW_COLOUR_2}Niveau d’intensité : {BLACK}Pas encore disponi
...
-STR_1519    :“Cette peluche de {STRINGID} est un très bon choix”
-STR_1520    :“Ce plan du parc de {STRINGID} est un très bon choix”
-STR_1521    :“Cette photo sur le vif de {STRINGID} est un très bon choix”
-STR_1522    :“Ce parapluie de {STRINGID} est un très bon choix”
-STR_1523    :“Cette boisson de {STRINGID} est un très bon choix”
-STR_1524    :“Cet hamburger de {STRINGID} est un très bon choix”
-STR_1525    :“Ces frites de {STRINGID} sont un très bon choix”
-STR_1526    :“Cette glace de {STRINGID} est un très bon choix”
-STR_1527    :“Cette barbe-à-papa de {STRINGID} est un très bon choix”
+STR_1480    :« Je ne peux pas me payer {STRINGID} »
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:869]: Are these regular spaces? It’s better to use non-breaking spaces, so you won't have «
situations like these ».

File: data/language/fr-FR.txt [hunk 2]
```diff
@@ -1828,8 +1828,8 @@ STR_2481    :Voir graphique Accélération latérale sur durée
 STR_2482    :Bénéfice : {CURRENCY} par semaine, {NEWLINE}Valeur du parc : {CURRENCY}
 STR_2483    :{WINDOW_COLOUR_2}Bénéfice hebdomadaire : {BLACK}+{CURRENCY2DP}
 STR_2484    :{WINDOW_COLOUR_2}Bénéfice hebdomadaire : {RED}{CURRENCY2DP}
-STR_2487    :Afficher le ‘vrai’ nom des visiteurs
-STR_2488    :Passer du ‘vrai’ nom des visiteurs à leur numéro
+STR_2487    :Afficher le « vrai » nom des visiteurs
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:1831]: This is not a quotation. Any reason why you gave this (and several other strings) guillemets, while other strings (e.g. 2742) got {SMALLFONT}?

File: data/language/fr-FR.txt (hunk 1 above)
Comment (rmnvgr) [reply data/language/fr-FR.txt:869]: They are non-breaking spaces.

File: data/language/fr-FR.txt (hunk 2 above)
Comment (rmnvgr) [reply data/language/fr-FR.txt:1831]: Yes, when a word should not be taken literally, you put guillemets around it.

**Comment by rmnvgr** [comment]
I've added [fr-FR translation guidelines](https://github.com/OpenRCT2/Localisation/wiki/Translator-guidelines:-fr_FR), and updated the translation to follow them.

**Comment by Gymnasiast** [comment]
Nice work! Sorry for taking so long to review it again.

Some things:
- You used a longer dash in some cases. This is fine if it is correct for French, but have you checked if OpenRCT2 displays it correctly? If not, I need to add support.
- The translation appears to use both `œ` and `oe`, inconsistently. AFAIK, `œ` is still proper French, is that correct? If so, I’d need to add a glyph for it, as we don’t currently have one.
- Some big changes are afoot for the objects translation, and I’d need to redump them. Since you didn’t yet do much to that (just easy substitutions), could you revert those changes for now? (Probably best to keep those changes somewhere just for reference)
- Could you rebase your pull request?

**Comment by rmnvgr** [comment]
> Nice work! Sorry for taking so long to review it again.

No worries, it's quite a substantial PR.

> You used a longer dash in some cases. This is fine if it is correct for French, but have you checked if OpenRCT2 displays it correctly? If not, I need to add support.

Indeed, I didn't check, and OpenRCT2 doesn't display it correctly.

> The translation appears to use both `œ` and `oe`, inconsistently. AFAIK, `œ` is still proper French, is that correct? If so, I’d need to add a glyph for it, as we don’t currently have one.

Good catch. `œ` is correct, I've changed the strings.

> Some big changes are afoot for the objects translation, and I’d need to redump them. Since you didn’t yet do much to that (just easy substitutions), could you revert those changes for now? (Probably best to keep those changes somewhere just for reference)

I've moved the changes to objects to [this branch](https://github.com/rmnvgr/Localisation/tree/improved-objects).

> Could you rebase your pull request?

Done!

File: data/language/fr-FR.txt [hunk 3]
```diff
@@ -714,25 +714,25 @@ STR_1324    :{COMMA16} parcours
...
+STR_1334    :{STRINGID} — {STRINGID} {COMMA16}
+STR_1335    :{STRINGID} — Entrée{POP16}{POP16}
+STR_1336    :{STRINGID} — Entrée de station {POP16}{COMMA16}
+STR_1337    :{STRINGID} — Sortie{POP16}{POP16}
+STR_1338    :{STRINGID} — Sortie de station {POP16}{COMMA16}
 STR_1339    :{BLACK}Pas encore de résultats de test…
 STR_1340    :{WINDOW_COLOUR_2}Vitesse maximale : {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Durée de l’attraction : {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} /
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:732]: This string and 1346 need the trailing whitespace. It’s not ideal, I know.

File: data/language/fr-FR.txt [hunk 4]
```diff
@@ -1116,7 +1116,7 @@ STR_1727    :Les droits de construction ne sont pas à vendre !
 STR_1728    :Impossible d’acheter les droits de construction ici…
 STR_1729    :Ce terrain n’appartient pas au parc !
 STR_1730    :{RED}Fermé
-STR_1731    :{WHITE}{STRINGID} - - 
+STR_1731    :{WHITE}{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:1119]: This string and 1778 need the trailing whitespace.

File: data/language/fr-FR.txt [hunk 5]
```diff
@@ -2197,20 +2197,20 @@ STR_3063    :Tunnel aquatique (voie engloutie)
...
-STR_3076    :{RED}DERNIER AVERTISSEMENT : l’évaluation de votre parc est toujours inférieure à 700 !{NEWLINE}Si elle n’a pas augmenté dans 7 jours, votre parc sera fermé
+STR_3073    :{RED}AVERTISSEMENT : l’évaluation de votre parc est inférieure à 700 !{NEWLINE}Si elle n’augmente pas d’ici à 4 semaines, votre parc sera fermé.
+STR_3074    :{RED}AVERTISSEMENT : l’évaluation de votre parc est toujours inférieure à 700 !NEWLINE}Il vous reste 3 semaines pour l’améliorer.
+STR_3075    :{RED}AVERTISSEMENT : l’évaluation de votre parc est toujours inférieure à 700 !{NEWLINE}Il ne vous reste plus que 2 semaines pour l’améliorer, sous peine de fermeture.
+STR_3076    :{RED}DERNIER AVERTISSEMENT : l’évaluation de votre parc est toujours inférieure à 700 !{NEWLINE}Si elle n’a pas augmenté dans 7 jours, votre parc sera fermé.
 STR_3077    :{RED}AVIS DE FERMETURE : votre parc a été fermé !
-STR_3090    :Choisir style d’entrée, de sortie et de station
-STR_3091    :Interdit de supprimer cette section !
-STR_3092    :Interdit de déplacer ou modifier la station de cette attraction !
+STR_3090    :Choisit  le style d’entrée, de sortie et de station
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:2211]: Double space.

```suggestion
STR_3090    :Choisit le style d’entrée, de sortie et de station
```

File: data/language/fr-FR.txt [hunk 6]
```diff
@@ -2354,26 +2354,26 @@ STR_3253    :Impossible de diminuer davantage l’emprunt maximum !
 STR_3254    :Impossible d’augmenter davantage le taux d’intérêt !
 STR_3255    :Impossible de diminuer davantage le taux d’intérêt !
 STR_3256    :Attr. moins dures pour visiteurs
-STR_3257    :Choisir si vous souhaitez que les visiteurs préfèrent des attractions moins intenses
+STR_3257    :Les visiteurs préfèrent des attractions moins intenses
 STR_3258    :Attr. plus dures pour visiteurs
-STR_3259    :Choisir si vous souhaitez que les visiteurs préfèrent des attr. plus intenses
-STR_3260    :{WINDOW_COLOUR_2}Argent par visiteur (moy.) :
+STR_3259    :Les visiteurs préfèrent des attractions plus intenses
+STR_3260    :{WINDOW_COLOUR_2}Argent par visiteur (en moyenne) :
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:2360]: This string might be too long for the window in question. Could you check it? (It’s the window in the Finances step of the Scenario Editor.)

File: data/language/fr-FR.txt [hunk 7]
```diff
@@ -2493,30 +2493,30 @@ STR_5125    :Tout destructible
...
+STR_5144    :Vitesse très rapide
+STR_5145    :Vitesse turbo
+STR_5146    :Vitesse hyper
 STR_5147    :Triche
 STR_5148    :Changer la vitesse du jeu
 STR_5149    :Ouvrir la fenêtre de triche
 STR_5150    :Activer les outils de débogage
 #Thousands separator
-STR_5151    : 
+STR_5151    :
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:2519]: This line needs to be reverted. (See the comment on the preceding line.)

File: data/language/fr-FR.txt [hunk 8]
```diff
@@ -2791,7 +2791,7 @@ STR_5463    :But : Amusez-vous !
 STR_5464    :Général
 STR_5465    :Météo
 STR_5466    :Personnel
-STR_5467    :ALT + 
+STR_5467    :ALT +
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:2794]: revert

File: data/language/fr-FR.txt [hunk 9]
```diff
@@ -2945,16 +2945,16 @@ STR_5621    :Loopings en Folie
...
 STR_5626    :Autres parcs
 STR_5627    :Grouper les scénarios par :
 STR_5628    :Jeu d’origine
 STR_5629    :Niveau de difficulté
 STR_5630    :Activer le déblocage progressif des scénarios
 STR_5631    :Parcs additionnels originaux
 STR_5632    :Construisez votre propre parc…
-STR_5633    :CMD + 
-STR_5634    :OPTION + 
+STR_5633    :CMD +
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:2956]: revert (and next line too)

File: data/language/fr-FR.txt [hunk 10]
```diff
@@ -3534,30 +3534,30 @@ STR_6272    :Quais
...
+STR_6276    :{RED}{STRINGID} a des visiteurs coincés, probablement à cause d’un type d’attraction invalide ou du mode d’opération
 STR_6277    :{WINDOW_COLOUR_2}Index du quai : {BLACK}{STRINGID}
 STR_6278    :Nombre de sauv. auto. :
 STR_6279    :Nombre de sauvegardes automatiques à conserver
 STR_6280    :Chat
 STR_6281    :Affiche un bouton séparé pour la fenêtre du chat dans la barre d’outils
 STR_6282    :Chat
-STR_6283    :Le chat n’est pas disponible actuellement. Etes-vous connecté(e) à un serveur ?
+STR_6283    :Le chat n’est pas disponible actuellement. Êtes-vous connecté(e) à un serveur ?
 STR_6293    :B
```
Comment (Gymnasiast) [inline data/language/fr-FR.txt:3545]: I thought French used `o` (octet) instead of `B` (bytes). Or did it switch to `B`?

**Comment by rmnvgr** [comment]
Sorry for the missing trailing spaces, my editor removes them automatically.

File: data/language/fr-FR.txt (hunk 10 above)
Comment (rmnvgr) [reply data/language/fr-FR.txt]: No, we use `o`, it just hadn't been translated.

File: data/language/fr-FR.txt (hunk 6 above)
Comment (rmnvgr) [reply data/language/fr-FR.txt:2360]: It fits in the window.

File: data/language/fr-FR.txt (hunk 5 above)
Comment (rmnvgr) [reply data/language/fr-FR.txt]: Good catch, it is fixed.

**Comment by Gymnasiast** [comment]
Good!

I have now added support for drawing the dashes and I also created sprites for the Œ ligature: https://github.com/OpenRCT2/OpenRCT2/pull/15034

**Comment by rmnvgr** [comment]
Nice. I kept the long dashes, I suppose a sprite is also needed for that?

**Comment by Gymnasiast** [comment]
> Nice. I kept the long dashes, I suppose a sprite is also needed for that?

I just added code to draw them as regular dashes. If needed/desired, we can add proper glyphs later on.

**Comment by rmnvgr** [comment]
Sounds good! 👍

**Pull request merged**